server/views/pages/debug/reportDetails.njk

```diff
@@ -113,7 +129,22 @@
     {%- set questionsRows = [] -%}
     {%- for question in report.questions -%}
       {%- if question.responses.length > 0 -%}
-        {%- set responses = question.responses | join('<br />', "response") -%}
+        {% set responses %}
```

Comment:
👍

Reply:
this is cos the date & comment might be worth showing